rhel8/profiles/ospp.profile

```diff
@@ -191,18 +191,12 @@ selections:
     ## Required Packages
     #################################################################
 
-    ## RHEL 8 CCE-82995-2: Install cryptsetup-luks Package
-    - package_cryptsetup-luks_installed
```

Comment:
Isn't this one needed to ensure hard drives are encrypted at rest with LUKS? 

Or rather, LUKS is the only FIPS 140-2 evaluated encryption at rest in RHEL. How do we ensure the system is configured with LUKS?

Reply:
I was told that OSPPv4.2(.1) doesn't have, and we don't claim, any HDD encryption, hence the package doesn't need to be installed.